docs/docs/setup/update.md

```diff
@@ -81,8 +81,11 @@ To update Meili:
     - ... that videos can be played.
     - ... that search works.
 
-
 :::note
 If you encounter a problem only when restarting the processes and you think it should have been caught by `tobira check`, let us know so that we can add that check!
 :::
 
+## Afterwards
+
+- *Conditional*: if necessary, run a [resync](./resync).
```

Comment:
Since this step has the caveat to it that it might take a while/stress the Opencast maybe it should be mentioned in the beginning somehow? :thinking:

Reply:
Mhhh... fair point. However, it doesn't need to be done right here and now. It's totally fine to just wait with this step until way after the update. Though then I would make it the second point in this section, not the first, yes.

Reply:
Fair enough. :shrug: